src/posts/dto/create-post.dto.ts

```diff
@@ -0,0 +1,7 @@
+export class CreatePostDto {
+  caption: string;
+  is_hidden: string;
+  user_id: number;
```

Comment:
Remove (user_id, ready & created) We don't send them in the request. We will set them up in postRepository file. Please revise the openAPI specs.